### PR TITLE
Target blank was not respectged when clicking on a new link

### PR DESCRIPTION
**Pull request by gerardnico**
This the PR for the issue #15 

I have added an HTML test page.

### CODE REVIEW

**Review by phy25 — CHANGES_REQUESTED** [review]
I think it was there to make sure Google Analytics finishes the job before redirecting. We can either use a different way to record it or check the target value, but this is not the right way to fix it I guess.

**Comment by gerardnico** [comment]
Ah yes you'r right. The original tab is also navigating. I will look into it later.

**Comment by gerardnico** [comment]
The callback is not needed. I have improved the test html page where I have written all steps to validate it.
I have also added a debug mode that prints the hit payload to the console.
This version is already running at https://datacadamia.com

**Comment by phy25** [comment]
My bad. Looks like when this code is initially introduced, `transport: beacon` is already used, and thus there is no need to wait for the request to complete.

I would recommend reusing DokuWiki-wide `$conf['allowdebug']` for it as I don't think it needs a separate control.

**Comment by gerardnico** [comment]
For what I see, they let just the default event takes place. They just record it. I have changed the local debug conf by the global one.

**Comment by phy25** [comment]
> they let just the default event takes place. They just record it

This is actually something called [Beacon API](https://developer.mozilla.org/en-US/docs/Web/API/Beacon_API) which does work even after users close the page. Note that it doesn't always work but it should be good enough.

File: lang/en/settings.php [hunk 1]
```diff
@@ -6,4 +6,5 @@
 $lang['track_users'] = 'Track logged in users across different devices. Needs <a href="https://support.google.com/analytics/answer/3123662?hl=en">user tracking</a> enabled. Users will not be identifable in Google Analytics!';
 $lang['track_links'] = 'Track outgoing links.';
 $lang['domainName'] = 'Configure the cookie domain. Should usually be left empty';
+$lang['debug'] = 'Debug mode: Send the hits to the console';
```
Comment (phy25) [inline lang/en/settings.php:9]: This line is not needed anymore.

```suggestion
```

**Review by phy25 — APPROVED** [review]

**Review by tatewake — APPROVED** [review]

**Comment by tatewake** [comment]
Hi @gerardnico and @phy25 , thank you for your changes and feedback.
Everything looks good here and I'll go ahead and prepare a new version release for the plugin.

**Comment by tatewake** [comment]
Hello; your changes have now been merged into master and there is a new release available if you wish to upgrade via the DokuWiki extension manager. Thanks again for your changes!

**Comment by gerardnico** [comment]
I have a little bit difficulty to follow the thread as I'm not used to github. The fact that you got a discussion on a PR and on an issue
is a little bit disturbing. Thanks for the feedback !

**Pull request merged**